clang/include/clang/Basic/AttrDocs.td

```diff
@@ -2741,6 +2778,12 @@ function object). If 1, compiler doesn't use the global work offset values for
 the device function. Valid values are 0 and 1. If used without argument, value
 of 1 is set implicitly.
 
+If ``intel::no_global_work_offset`` is applied to a function called from a
+device kernel, the attribute is not ignored and it is propagated to the kernel
+in SYCL 1.2.1 spec. SYCL 2020 spec does not allow propagating the attribute if
+``intel::no_global_work_offset`` is applied to a function called from a
+device kernel.
+
```

Comment:
```suggestion
When conforming to SYCL 1.2.1 specifications, when called from a device kernel, the ``intel::no_global_work_offset`` is propagated from the function it is applied to, to the kernel.  When conforming to SYCL 2020 specifications, the attribute is not propagated to the kernel.```

Reply:
I feel this works better.   If you agree, I would like this phrasing used for the other changes too.

Reply:
Done